docs/source/release_notes/release-notes.rst

```diff
@@ -72,6 +72,9 @@ develop
          - The priority of logging on background sweep was increased from debug to info or warn.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2031>`__)
 
+    *    - |improved|
+         - Sweep now is capable of dynamically adjust the number of blocks - (cell, ts) pairs - per run.
```

Comment:
Sweep does halve the number of parameters at every failed run. Internally, we were calling this 'dynamic adjustment'. I'll clarify the release note.